core/services/offchainreporting2/database_test.go

```diff
@@ -134,12 +134,12 @@ func Test_DB_ReadWriteConfig(t *testing.T) {
 	config := ocrtypes.ContractConfig{
 		ConfigDigest:          testhelpers.MakeConfigDigest(t),
 		ConfigCount:           1,
-		Signers:               []ocrtypes.OnchainPublicKey{},
-		Transmitters:          []ocrtypes.Account{"account1"},
+		Signers:               []ocrtypes.OnchainPublicKey{{0x01}, {0x02}},
```

Comment:
note on 159 we assert written config equals what we read back, so adding this gives us test coverage